src/components/tooltip-v2/migrate-to-7.x.md

```diff
@@ -0,0 +1,25 @@
+### HTML
```

Comment:
Given V2 is newly introduced component, does this migration guide still make sense? One thing that comes to my mind is V1 to V2 migration guide. Another approach may be simply keeping the README in-sync with what V2 does.